CMakeLists.txt

```diff
@@ -113,19 +113,24 @@ if(HUNTER_ENABLED)
   hunter_add_package(Protobuf)
   find_package(Protobuf ${hunter_config} REQUIRED)
   list(APPEND package_deps Protobuf)
-  set(protoc_exe protobuf::protoc)
-else()
```

Comment:
So how do we generate the messages? How do we connect to backend at all?

Reply:
You will have seen the answer to that further down, guessing this is outdated. It's just a change in how the IDL is compiled by cmake.